### DISCUSSION TITLE
"Object Selection" select/unselect all

### DISCUSSION THREAD

**Discussion by luotiansha** [OpenRCT2/OpenRCT2#21881] [Feature requests]
Make it more convenient to select all / unselect all, so that one doesn't need an auto clicker

![屏幕快照 2024-04-23 的 06 17 01 上午](https://github.com/OpenRCT2/OpenRCT2/assets/77926812/27a14359-8719-4811-9d54-2231627d3b83)

**Comment by MASHzj** [comment]
Duplicate of #15343
